ui/app/components/ui/list-item/list-item.stories.js

```diff
@@ -0,0 +1,60 @@
+import React from 'react'
+import ListItem from './list-item.component'
+import { text } from '@storybook/addon-knobs/react'
+
+export default {
+  title: 'ListItem',
+}
+
+const More = () => (
+  <span>
+    <button className="list-item__more--button">{text('Button1', 'Speed up')}</button>
```

Comment:
Nit: it doesn't really matter if this class is to be removed, but technically this isn't a valid BEM modifier either, e.g. it should be `list-item__more-button`